Zend/tests/property_hooks/gh19548.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+GH-19548: SEGV do_inherit_property zend_inheritance.c with opcache.protect_memory=1
+--EXTENSIONS--
+opcache
+--INI--
+opcache.enable_cli=1
+opcache.protect_memory=1
```

Comment:
Should be moved to ext/opcache/tests, or this section should be removed entirely. Tests are ran with and without opcache, and `opcache.protect_memory=1` is always enabled by run-tests.php.